README.md

```diff
@@ -91,6 +91,20 @@ bazel
 
 Bazelisk itself should not require further updates, as its only purpose is to keep Bazel updated.
 
+## zlib on Linux
```

Comment:
does any of the OS distribution include this by default ?  if they do, then customer doesnt need to download and compiler.

actually, how can a developer check what zlib is running ? in fullness of time, if all OSes adopt cloudflare, then this step can be avoided

Reply:
I'll update with that info.

Reply:
@AWSNB I've added a step to determine which zlib is being used.

Reply:
None do today, this just takes up space. I'd just say, zlib packaged by distros doesn't include these optimizations. If you'd like to use them follow these steps and if that ever changes we update the doc.